tools/db_bench_tool.cc

```diff
@@ -4691,6 +4741,8 @@ class Benchmark {
     return FLAGS_sine_a*sin((FLAGS_sine_b*x) + FLAGS_sine_c) + FLAGS_sine_d;
   }
 
+  void WriteOrSelectiveDelete();
```

Comment:
Undefined / unused?

Reply:
I am cleaning that up right now - I cant believe the armada of warnings/compilers didn't catch this.